app/api/auth.py

```diff
@@ -269,3 +273,24 @@ def change_password():
         "name": user.fullname if user.fullname else None,
         "password-changed": True
     })
+
+
+def authorized_access_ticket(ticket_function):
+    @wraps(ticket_function)
+    def wrapper(*args, **kwargs):
+        try:
+            user_id = Order.query.filter_by(identifier=kwargs['identifier']).first().user.id
```

Comment:
Don't just catch a generic exception. Just check if it returns none and send 404.